components/user/profile.tsx

```diff
@@ -1,13 +1,14 @@
 'use client';
 
-import { TimeDiff } from '@/shared/time-diff';
-import { UserImage } from '@/shared/user-image';
+import { TimeDiff } from '@/components/_shared/time-diff';
+import { UserImage } from '@/components/_shared/user-image';
 import { Location } from '@/user/location';
 import { UserHandle } from '@/user/user-handle';
 import { getName } from '@/user/user-utils';
 import { User } from '@/utils/models';
-import { EditIcon, SettingsIcon } from 'clada-storybook';
+import { EditIcon } from 'clada-storybook';
 import NextImage from 'next/image';
+import { SettingsModal } from '../modals/settings';
 
 const fakeULID = '01HGX50H2SXXPE51S60Q7QY29M';
```

Comment:
Do we need this fakeULID still?

Reply:
The design presents an element in the user profile that shows how long the user has been registered (Mitglied seit 4 Wochen):
![image](https://github.com/ost-cas-fee-adv-23-24/nextjs-app-clada/assets/144836241/93bca350-15a7-4cd5-be6a-eeb893d08565)

The problem is that the userId is not an ULID and there is no way to get at this date. That is why i am faking it here. Much like we are faking the user profile background image (that cannot be stored or retrieved via API).

I have added a comment

Reply:
Alright, makes sense then. 👍🏻